### ISSUE TITLE
Set Host header for readiness probe

### ISSUE THREAD

**Issue by llnz** [bootc/netbox-chart#14]
If Allowed Hosts is not '*', then the readiness probe always fails with 500 Server Error (and emails the admin). This can be avoided by setting the Host header on the readiness probes, such as:

```yaml
     readinessProbe:
            httpGet:
              path: /
              port: http
              httpHeaders:
                - name: Host
                  value: hostname.example.com
```

The value for the header could be the first value in the allowedHosts list.

**Comment by bootc** [comment]
This is fixed in chart version `2.1.0`, which includes NetBox 2.8.1 (but can support NetBox 2.7.11 or above).

**Comment by llnz** [comment]
This needs to be applied to the nginx container readinessProbe as well, as the request is forwarded onto the app.
